.eslintrc.cjs

```diff
@@ -5,12 +5,46 @@ module.exports = {
     'eslint:recommended',
     'plugin:@typescript-eslint/recommended',
     'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'plugin:prettier/recommended',
     'prettier',
   ],
   ignorePatterns: ['dist', '.eslintrc.cjs'],
```

Comment:
```suggestion
  ignorePatterns: ['dist'],
```
This is a bit odd. ESLint doesn’t lint config files by default unless explicitly told to, so you can remove this unless there's a specific reason